src/Hl7.Fhir.Specification.Tests/Source/SummarySourceTests.cs

```diff
@@ -147,11 +147,7 @@ public async Tasks.Task TestThreadSafety()
                 tasks[i] = Tasks.Task.Run(
                     () =>
                     {
-#if DOTNETFW
                         var threadId = System.Threading.Thread.CurrentThread.ManagedThreadId;
```

Comment:
Idem

Reply:
Let op: dit is een unittest en draait nu op netcoreapp2.1. En daar zit dit zeker in